test-clients/src/main/java/com/hedera/services/bdd/suites/perf/MixedOpsLoadTest.java

```diff
@@ -0,0 +1,285 @@
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_NOT_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOPIC_EXPIRED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_EXPIRED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNKNOWN;
+
+public class MixedOpsLoadTest extends LoadTest {
+	private static final Logger log = LogManager.getLogger(MixedOpsLoadTest.class);
+	private static final int NUM_SUBMISSIONS = 100;
+	private final ResponseCodeEnum[] permissiblePrechecks = new ResponseCodeEnum[] {
+			BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED, UNKNOWN
+	};
+
+	private static String sender = "sender";
+	private static String receiver = "receiver";
+	private static String submitKey = "submitKey";
+	private static String topic = "topic";
+	private static String token = "token";
+	private static String schedule = "schedule";
+	private static int messageSize = 1024;
+
+	public static void main(String... args) {
+		parseArgs(args);
+		MixedOpsLoadTest suite = new MixedOpsLoadTest();
+		suite.setReportStats(true);
+		suite.runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				runMixedOps()
+		);
+	}
+
+	@Override
+	public boolean hasInterestingStats() {
+		return true;
+	}
+
+	protected HapiApiSpec runMixedOps() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+		Random r = new Random();
+		AtomicInteger tokenId = new AtomicInteger(0);
+		AtomicInteger scheduleId = new AtomicInteger(0);
+
+		Supplier<HapiSpecOperation[]> mixedOpsBurst = () -> new HapiSpecOperation[] {
+				cryptoTransfer(
+						tinyBarsFromTo(sender, receiver, 1L))
+						.noLogging()
+						.payingWith(sender)
+						.signedBy(GENESIS)
+						.suppressStats(true)
+						.fee(ONE_HBAR)
+						.hasKnownStatusFrom(SUCCESS,
+								OK,
+								INSUFFICIENT_PAYER_BALANCE,
+								UNKNOWN,
+								TRANSACTION_EXPIRED)
+						.hasRetryPrecheckFrom(BUSY,
+								DUPLICATE_TRANSACTION,
+								PLATFORM_TRANSACTION_NOT_CREATED,
+								PAYER_ACCOUNT_NOT_FOUND)
+						.hasPrecheckFrom(INVALID_SIGNATURE)
+						.deferStatusResolution(),
+
+				submitMessageTo(topic)
+						.message(ArrayUtils.addAll(ByteBuffer.allocate(8).putLong(Instant.now().toEpochMilli()).array(),
+								randomUtf8Bytes(messageSize - 8)))
+						.noLogging()
+						.payingWith(GENESIS)
+						.signedBy(sender, submitKey)
+						.fee(ONE_HBAR)
+						.suppressStats(true)
+						.hasRetryPrecheckFrom(BUSY,
+								DUPLICATE_TRANSACTION,
+								PLATFORM_TRANSACTION_NOT_CREATED,
+								TOPIC_EXPIRED,
```

Comment:
If a topic is expired, can it be brought back?

Reply:
@ljianghedera I am not sure. But we made multiple changes to the clients in order to resolve errors. I took the precheks from `SubmitMessageLoadTest`

Reply:
@Neeharika-Sompalli, @ljianghedera, we could create different lists of known status, of retry precheck, of precheck for different kind of perf ops (cryptoTransfer, submitMessage...) then share these lists among different test suites.

Reply:
Same here.

Reply:
@qnswirlds that's a good idea. We can create lists for commonly used response status code, then let the developer to add whatever status code for his particular tests.